docs/adding games.md

```diff
@@ -142,19 +114,13 @@ anything with it.
   
 ### What Modifications You Should Make to the Game
 We talked about this briefly in [Game Modification](#game-modification) section.
-The next step is to know what you need to make the game do now that you can modify it. Here are your key goals:  
-- Modify the game so that checks are shuffled  
-- Know when the player has completed a check, and react accordingly  
-- Listen for messages from the Archipelago server  
-- Modify the game to display messages from the Archipelago server  
-- Add interface for connecting to the Archipelago server with passwords and sessions  
-- Add commands for manually rewarding, re-syncing, releasing, and other actions  
+The next step is to know what you need to make the game do now that you can modify it. Here are your key goals:
+- Know when the player has checked a location, and react accordingly
+- Be able to receive items from the server on the fly
+- Keep an index for items received in order to resync from disconnections
+- Add interface for connecting to the Archipelago server with passwords and sessions
+- Add commands for manually rewarding, re-syncing, forfeiting, and other actions
   
-To elaborate, you need to be able to inform the server whenever you check locations, print out messages that you receive
-from the server in-game so players can read them, award items when the server tells you to, sync and re-sync when necessary,
-avoid double-awarding items while still maintaining game file integrity, and allow players to manually enter commands in
-case the client or server make mistakes. 
-
 Refer to the [Network Protocol documentation](./network%20protocol.md) for how to communicate with Archipelago's servers.  
   
 ## But my Game is a console game. Can I still add it?  
```

Comment:
Rather than "Can I Still Add it?", which sounds like you are asking permission, "How Easy will it be to add?" is kinder and does not suggest we will limit the games being added. Some of the sections below should probably be adjusted to reflect this

Reply:
untouched by this PR. goal was to remove redundant information and fix blatant inaccuracies.